common/src/main/java/org/jboss/metadata/parser/util/XMLResourceResolver.java

```diff
@@ -191,6 +202,28 @@ public Object run() {
         registerEntity("http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd", "web-app_2_5.xsd");
         registerEntity("http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd", "web-app_3_0.xsd");
         registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd", "web-app_3_1.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd", "web-app_4_0.xsd");
+        registerEntity("https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd", "web-app_5_0.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-common_3_0.xsd", "web-common_3_0.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-common_3_1.xsd", "web-common_3_1.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-common_4_0.xsd", "web-common_4_0.xsd");
+        registerEntity("https://jakarta.ee/xml/ns/jakartaee/web-common_5_0.xsd", "web-app_5_0.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-facelettaglibrary_2_2.xsd", "web-facelettaglibrary_2_2.xsd");
+        registerEntity("http://xmlns.jcp.org/xml/ns/javaee/web-facelettaglibrary_2_3.xsd", "web-facelettaglibrary_2_3.xsd");
+        registerEntity("https://jakarta.ee/xml/ns/jakartaee/web-facelettaglibrary_5_0.xsd", "web-facelettaglibrary_5_0.xsd");
```

Comment:
I think this one should refer to `3.0` instead of `5.0` (web-facelettaglibrary_3_0.xsd)

Reply:
Fixed.